src/js/components/pages.js

```diff
@@ -78,12 +78,27 @@ class Page {
     ];
   }
 
-  renderProfileImage() {
+  /**
+   *
+   * @param {String} [imgType] - One of profile or thumbnail - [default: thumbnail]
+   */
+  renderProfileImage(imgType) {
     const elements = [];
+
     if (this.profileImage) {
       const imageUrl = this.profileImage.meta.download_url;
       const imageAlt = this.profileImage.title;
-      elements.push($(`<img src="${imageUrl}" alt="${imageAlt}"></img>`));
+
+      if (imgType === "profile") {
```

Comment:
when would we not want rounded corners for this component? I don't think we need this logic - I think you can assume rounded always

Reply:
I was thinking we might only want it on the individual person pages. Let me test when I blanket apply.

Reply:
Turns out those use a `link-block` anyhow so, `renderProfileImage` is only for the individual profile pages 🤦‍♂️ 😃